File: 29a/Nutrizionista.py
from Dipendenti import Dipendenti
class Nutrizionista(Dipendenti):
    def __init__(self, cod, tipo, nome, costoOrario,telefono,medico,appuntamentiSettimanali) -> None:
        super().__init__(cod, tipo, nome, costoOrario)
        self.telefono = telefono
        self.medico   = medico
        self.appuntamentiSettimanali = appuntamentiSettimanali

    def __str__(self) -> str:
        return self.nome+"\t"+str(self.cod)+"\t"+self.tipo+"\t"+"-"+"\t"+"-"+"\t"+str(self.telefono)+"\t"+self.medico+"\t"+str(self.appuntamentiSettimanali)+str(self.costoOrario)

File: Sim05 -Cliente Dipendente Nutrizionista Trainer/src/Nutrizionista.java
public class Nutrizionista extends Dipendente {
    long telefono;
    Boolean medico;
    int appuntamenti_settimanali;


    public Nutrizionista(int codice, double costo_orario, String tipo, String nome_dipendente, long telefono,
            Boolean medico, int appuntamenti_settimanali) {
        super(codice, costo_orario, tipo, nome_dipendente);
        this.telefono = telefono;
        this.medico = medico;
        this.appuntamenti_settimanali = appuntamenti_settimanali;
    }




    public String toString() {
        return nome_dipendente+" " + codice+" " + tipo + " - " + " - " + telefono+" " + medico+" " + appuntamenti_settimanali+" " + costo_orario;
    }

}
